Ошибка: HTTPConnectionPool(host='p', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x00000161C5034700>: Failed to resolve 'p' ([Errno 11001] getaddrinfo failed)"))
URL: http://p